debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -757,12 +757,13 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
             .withWidth(Width.SHORT)
             .withImportance(Importance.LOW)
             .withDescription("The criteria for running a snapshot upon startup of the connector. "
-                    + "Options include: "
-                    + "'when_needed' to specify that the connector run a snapshot upon startup whenever it deems it necessary; "
-                    + "'schema_only' to only take a snapshot of the schema (table structures) but no actual data; "
-                    + "'initial' (the default) to specify the connector can run a snapshot only when no offsets are available for the logical server name; "
-                    + "'initial_only' same as 'initial' except the connector should stop after completing the snapshot and before it would normally read the binlog; and"
-                    + "'never' to specify the connector should never run a snapshot and that upon first startup the connector should read from the beginning of the binlog. "
+                    + "Select one of the following snapshot options: "
+                    + "'when_needed': On startup, the connector runs a snapshot if one is needed.; "
+                    + "'schema_only': If the connector does not detect any offsets for the logical server name, it runs a snapshot that captures only the schema (table structures), but not any table data.; "
+                    + "'schema_only_recovery': The connector performs a snapshot that captures only the database schema history. The connector then transitions back to streaming. Use this setting to restore a corrupted or lost database schema history topic. Do not use if the database schema was modified after the connector stopped.; "
+                    + "'initial' (default): If the connector does not detect any offsets for the logical server name, it runs a snapshot that captures the current full state of the configured tables. After the snapshot completes, the connectors begins to stream changes from the binlog.; "
+                    + "'initial_only': The connector performs a snapshot as it does for the  'initial' option, but after completing the snapshot, it does not stream changes from the binlog.; "
+                    + "'never': The connector does not run a snapshot. Upon first startup, the connector immediately begins reading from the beginning of the binlog. "
```

Comment:
Similar to the others, the wording about transitioning to streaming and capturing changes from the binlog apply to all these modes except `initial_only`, and yet we only clarify this for `initial_only`.  Is there a way we could reword this to be clear?